### DISCUSSION TITLE
Automatic rollback ignores setting

### DISCUSSION THREAD

**Discussion by p91paul** [dbeaver/dbeaver#12100] [Q&A]
#### System information: 
- Operating system (distribution) and version Windows 10
- DBeaver version 21.0.2.202104042040
- Additional extensions n/a

#### Connection specification:
- Database name and version Oracle 11
- Driver name ojdbc 19
- Do you use tunnels or proxies (SSH, SOCKS, etc)? n/a

#### Describe the problem you're observing:
Despite the new option for automatic rollback of idle transactions being off (I checked on the connection specific settings as well), I today got a notification that my transaction was automatically rolled back. Which is fine mine being a select-only transaction, but still not what I expected.
Looking to query manager, looks like rollback happened 15 minutes (900s) after my last query, so exactly like the option was checked.

![immagine](https://user-images.githubusercontent.com/2157006/114401112-e55cbf00-9ba2-11eb-97f9-faf66f5f8814.png)

As a side note, that settings dialog should tell somewhere that 900 is a number of seconds, there is no measurement unit.

**Comment by flambchap** [comment]
Hi @p91paul. Do you use the `production` connection type for your connection? If that's the case, you need to uncheck the corresponding option in connection type settings (focus on navigator -> f4 -> `General` -> `Edit connection types`).

**Comment by p91paul** [comment]
Will check that, thanks. I still believe it's wrong to turn that option on by default on any connection type.

**Comment by p91paul** [comment]
That was it.
Your call whether to restore the previous default or close this.

**Comment by n8falke** [comment]
I've the same problem even for some weeks in Production connection type. And it has nothing to do with the setting "900" with I did not enable or change. Today the Rollback was issued after 152 min 49 sec. Im using Oracle 12.1.
